index_datas/import_datas.py

```diff
@@ -0,0 +1,140 @@
+
+import pandas as pd
+import requests
+import json
+import gzip
+from lxml import etree
+
+import wikipedia, gzip, threading
+from urllib.request import urlopen
+from elasticsearch import Elasticsearch as es
+from wikipedia.exceptions import PageError, DisambiguationError
+import ssl
+from multiprocessing import Pool
+import multiprocessing
+
+from elasticsearch import Elasticsearch as es
+elastic_search = es(["127.0.0.1"], timeout=35, max_retries=8, retry_on_timeout=True)
```

Comment:
**[Question, Maintainability]** So there is this static declaration for an elasticsearch client connection and there is also the `start_new_elasticsearch()` method, if static declarations are going to be used, then should make use of that method.

Reply:
ok